src/app/containers/MediaPlayer/helpers/logMissingBlockId.js

```diff
@@ -0,0 +1,12 @@
+import nodeLogger from '#lib/logger.node';
+import { MEDIA_PLAYER_STATUS } from '#lib/logger.const';
+
+const logger = nodeLogger(__filename);
+
+const logMissingBlockId = ({ url, assetType }) => {
+  const blockIdError = 'Missing Block ID';
```

Comment:
I don't know what our convention is for error messages

Reply:
No real strong convention - usually just something like this:

```javascript
logger.warn(MEDIA_PLAYER_STATUS, {
  message,
  url,
});
```